src/scripts/utils/AppUtils.js

```diff
@@ -51,8 +51,8 @@ export const getParatiiConfig = (env: ?string = 'development'): Object => {
 }
 
 export const prettyBytes = (num: ?number): string => {
-  if (num === null || num === undefined) {
-    num = 0
+  if (num === null || num === undefined || num === 0 || isNaN(num)) {
```

Comment:
Why check if `num === 0`? Wouldn't we want to print `0B` or something then?

Can this be simplified to `if (typeof num !== 'number' || isNaN(num))`?